flink-examples/flink-examples-build-helper/flink-examples-streaming-gcp-pubsub/pom.xml

```diff
@@ -29,7 +29,7 @@ under the License.
 	</parent>
 
 	<artifactId>flink-examples-streaming-gcp-pubsub_${scala.binary.version}</artifactId>
-	<name>flink-examples-streaming-gcp-pubsub</name>
+	<name>Flink : Examples : Dist : Streaming Google PubSub</name>
```

Comment:
Dist -> Helper?

Reply:
Yes, can be. I had doubts about this one. The artifact name is build helper yet the purpose is making it available in the dist.

Reply:
For now I made it consistent with the module artifact name : "Build Helper".

Reply:
I personally would rename this to something like "dist helper" or "dist examples" but that is a separate change if desired.